openebs/buildscripts/docker/Dockerfile

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:16.04
+ARG BASE_IMAGE=ubuntu:16.04
```

Comment:
Not required to set default again, as we are already set the `BASEIMAGE` defaults in Makefile
we can just change it to:
`ARG BASE_IMAGE=ubuntu:16.04` ---> `ARG BASE_IMAGE`

Reply:
this is just a good practice to set default value. in case the env isn't working or someone tries to directly call the docker command.